CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+# Catch-all
+* @tgraf @manalibhutiyani
+
+# BPF Data Path
+bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
+pkg/maps/* @joestringer @borkmann @jrfastab @manalibhutiyani
+pkg/bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
+# TODO Where are "KProxy kernel" and "XDP Prefilter" parts?
+
+# Agent -> Kernel interaction
+# TODO No idea where is it
+# @scanf @borkmann
+
+# CI
+tests/* @ianvernon @eloycoto
+Jenkinsfile @ianvernon @eloycoto
+
+# KVStore
+pkg/kvstore/* @ianvernon @aanm @tgraf @rlenglet
+
+# K8s integration, K8s CNI plugin
+daemon/k8s_watcher.go @aanm @nebril
+pkg/k8s/* @aanm @nebril
+plugins/cilium-cni/* @aanm @nebril
+
+# Docker plugin
+plugins/cilium-docker/* @nebril @manalibhutiyani
+
+#TODO: If mesos plugin is created: @nebri @manalibhutiyani @aanm
+
+# Policy behaviour
+pkg/policy @nebril @jrajahalme @manalibhutiyani
+
+# Agent API
+api/* @rlenglet @ianvernon @manalibhutiyani
+pkg/policy/api/* @rlenglet @ianvernon @manalibhutiyani
+pkg/apierror/* @rlenglet @ianvernon @manalibhutiyani
+pkg/apisocket/* @rlenglet @ianvernon @manalibhutiyani
+
+# Packaging
+contrib/packaging/deb/* @eloycoto
+contrib/packaging/docker/* @aanm @ianvernon
+
+# Proxy
+pkg/proxy/* @jrajahalme @manalibhutiyani @raybejjani
+# TODO: where is Envoy code located? same as above
+# TODO: where is kproxy located? @jrfastab @manalibhutiyani
+# TODO: pkg/proxy/accesslog/* owner needed
```

Comment:
I recommend @rlenglet and @manalibhutiyani.

Reply:
Yes, please add @rlenglet, @ashwinp, and @manalibhutiyani.